src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexNode.cs

```diff
@@ -865,6 +839,11 @@ internal SymbolicRegexNode<S> MkDerivative(S elem, uint context)
         }
 
         public override int GetHashCode()
+        {
+            return _hashcode;
+        }
+
+        private void EnsureHashCode()
```

Comment:
Since we're not always calling this in the ctor, can this instead be `private int ComputeHashCode();` and then the field can be `private readonly int _hashcode;` and the ctor can do `_hashcode = ComputeHashCode();`?

Reply:
yes, was considering that actually, will do